test/rekt/features/apiserversource/data_plane.go

```diff
@@ -318,7 +318,8 @@ func SendsEventsWithEventTypes() *feature.Feature {
 		Must("ApiServerSource test eventtypes match",
 			eventtype.WaitForEventType(
 				eventtype.AssertReady(expectedCeTypes),
-				eventtype.AssertPresent(expectedCeTypes)))
+				eventtype.AssertPresent(expectedCeTypes),
+				eventtype.AssertReferencePresent(broker.AsKReference(brokerName))))
```

Comment:
```suggestion

```
Technically, this should reference the sink instead of the broker (since that's how the APIServerSource creates it's eventtypes). However, there is a bug that I found thanks to this test: https://github.com/knative/eventing/issues/7895

So, for now lets remove this line and then merge the existing changes.